app/styles/_overview.less

```diff
@@ -81,6 +81,9 @@
       }
     }
   }
+  .service-binding h3 small {
```

Comment:
Yeah I originally called it `.application-binding` and forgot to move when I renamed... Will fix.